### ISSUE TITLE
In list of notifications, links to a specific post don’t jump directly to that post in feed

### ISSUE THREAD

**Issue by bfackland** [ploneintranet/ploneintranet#850]
Notification links just link to root URL - no anchors or markup for specific posts in social feed.

![screen shot 2015-10-22 at 11 24 25](https://cloud.githubusercontent.com/assets/2538526/10662962/cfab83d8-78af-11e5-849a-a976c7260995.png)

**Comment by mattss** [comment]
Please note - this is currently deliberate, as individual posts do not currently have their own URLs.

**Comment by bfackland** [comment]
Would it not be possible to at least reference them by anchor using some kind of UID?

**Comment by mattss** [comment]
@bfackland potentially, but there is no guarantee that they actually appear by default in a given stream (e.g. they may be too old)

**Comment by bfackland** [comment]
True. I guess we shouldn’t be showing notifications in the popup for more than x days ago anyway :)

I would have thought that most of the time the post to which the notification relates will be visible on the page, as it will have been fairly recent. IMO, better than having the somewhat useless links that are there now.

> On 22 Oct 2015, at 15:36, Matthew Sital-Singh notifications@github.com wrote:
> 
> @bfackland potentially, but there is no guarantee that they actually appear by default in a given stream (e.g. they may be too old)
> 
> —
> Reply to this email directly or view it on GitHub.

## 

Ben Ackland, Projects Director
Netsight Internet Solutions Limited
Tel: +44 (0)117 90 90 90 1

Registered in England no. 3892180
Registered office: 40 Berkeley Square, Bristol, BS8 1HU, United Kingdom

**Comment by mattss** [comment]
This is now waiting on https://github.com/ploneintranet/ploneintranet/issues/486
